Sources/NostrSDK/Keys.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  Keys.swift
+//  
+//
+//  Created by Terry Yiu on 5/23/23.
+//
+
+import Foundation
+import secp256k1
+
+struct Keypair {
+    public let publicKey: PublicKey
+    public let privateKey: PrivateKey
+
+    init?() {
+        guard let key = try? secp256k1.Signing.PrivateKey() else {
+            print("Could not generate new secp256k1 private key.")
```

Comment:
Maybe it makes sense to not swallow any of the errors in this file but re-throw? What do you think?

Reply:
If this is the kind of thing that never actually fails, maybe it's unnecessary to burden the call sites.